portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,20 +32,52 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
```

Comment:
consider adding class comment to explain the purposes of these classes.

The name Task is too generic. Please consider making it more descriptive.